src/services/OfferService.js

```diff
@@ -7,7 +7,11 @@ import createToken from '../utils/createToken'
 import MailService from './MailService'
 import forms from '../models/forms'
 
-const MILLISECONDS_IN_A_DAY = 1000 * 60 * 60 * 24
+const MS_IN_S = 1000
+const S_IN_MIN = 60
+const MIN_IN_H = 60
+const H_IN_D = 60
+const MILLISECONDS_IN_A_DAY = MS_IN_S * S_IN_MIN * MIN_IN_H * H_IN_D
```

Comment:
Imo this is so much uglier than it could be :D but oh well^^

Reply:
We can discuss the linting rules, BUT: This makes clear that a library for time should be used here which provides a type for a duration.